packages/client/src/pages/CheckerPage.js

```diff
@@ -162,7 +164,7 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
       newQuestionIndex = value;
       // Matomo event props
       action = actions.EDIT_QUESTION;
-      eventName = checker.stack[newQuestionIndex].text;
+      eventName = checker.stack[newQuestionIndex]?.text || t("common.unknown");
```

Comment:
This is strange, IMHO, because on line 173 (`action = checker.stack[questionIndex].text`) it is not checked liked this.

Is the reason we're doing it like this because we get errors with this? In that case it's better to check if `checker.stack[newQuestionIndex]` exists in the first place. Because then we don't need events like this, right? Shall we discuss this tomorrow?
 
It would be better to not to call the whole event if the new index doesn't exist. Now we're calling a

Reply:
Let's send a error to sentry if newQuestionIndex is not found.